### PR TITLE
Fix potential segfault in TransfersManager::addTransfer

### PR DESCRIPTION
**Pull request by fnkkio**

### CODE REVIEW

File: src/core/Updater.cpp [hunk 1]
```diff
@@ -59,9 +59,12 @@ Updater::Updater(const UpdateChecker::UpdateInformation &information, QObject *p
 	downloadFile(information.scriptUrl, path);
 
 	m_transfer = downloadFile(information.fileUrl, path);
-	m_transfer->setUpdateInterval(500);
+	if (m_transfer)
```
Comment (Emdek) [inline src/core/Updater.cpp]: Needs empty line before `if`.

File: src/core/Updater.cpp [hunk 2]
```diff
@@ -130,9 +133,12 @@ Transfer* Updater::downloadFile(const QUrl &url, const QString &path)
 {
 	const QString urlString(url.path());
 	Transfer *transfer(TransfersManager::startTransfer(url, path + urlString.mid(urlString.lastIndexOf(QLatin1Char('/')) + 1), (Transfer::CanOverwriteOption)));
-	transfer->setParent(this);
+	if (transfer)
```
Comment (Emdek) [inline src/core/Updater.cpp]: Same here, there should be always a blank line before and after `if...else` block etc., unless it is next to start or end of parent block.

**Comment by fnkkio** [comment]
@Emdek Added missing newlines.

**Comment by Emdek** [comment]
@fnkkio, great. :-)
Just one minor thing left, please squash the last two commits into one, since these two fixes should be kept separate.

**Comment by fnkkio** [comment]
@Emdek squashed.

**Pull request merged**